docs/application/dotnet/guides/user-interface/nui/layouts.md

```diff
@@ -14,15 +16,24 @@ In addition, the layout framework allows nested layouts. `View` with a layout ca
 
 A `View` can be assigned to any layout and then can be re-assigned to a different layout. However, a layout can be used with one `View` at a time.
 
-New layouts can be implemented by deriving them from the layout base class and positioning the children according to the requirement. To create a custom layout, see [Creating custom Layouts](./custom-layout.md).
+New layouts can be implemented by deriving them from the layout base class and positioning the children according to the requirement. To create a custom layout, see [Creating Custom Layouts](./custom-layout.md).
+
+## Common layouts
 
+| Type | Description | Look |
+| --- | --- | --- |
+| [AbsoluteLayout](./absolute-layout.md) | `AbsoluteLayout` is View's default layout that allows explicit positioning of child elements. The positions of children start from the top left of the layout and can be set by using the `Position` property.<br /><br /> [Guide](./absolute-layout.md) / [API Documentation](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.AbsoluteLayout.html) / [C# code](./nui-example-code/AbsoluteLayoutExample.cs) / [XAML code](./nui-example-code/AbsoluteLayoutExample.xaml) | ![AbsoluteLayout Example](./media/absolute-Layout.png) |
```

Comment:
A similar change was suggested by @mihashco back on Oct 07, 2021, but was not incorporated. 
```suggestion
| [AbsoluteLayout](./absolute-layout.md) | `AbsoluteLayout` is the default `View` layout that allows explicit positioning of child elements. The position of children starts from the top of the layout and can be set using the `Position` property.<br /><br /> [Guide](./absolute-layout.md) / [API Documentation](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.AbsoluteLayout.html) / [C# code](./nui-example-code/AbsoluteLayoutExample.cs) / [XAML code](./nui-example-code/AbsoluteLayoutExample.xaml) | ![AbsoluteLayout Example](./media/absolute-Layout.png) |
```